internals/notifier.py

```diff
@@ -864,6 +866,43 @@ def build_email(
     }
 
 
+class IntentToBlinkDevHandler(basehandlers.FlaskHandler):
+  """Submit an intent email directly to blink-dev."""
+  IS_INTERNAL_HANDLER = True
+  EMAIL_TEMPLATE_PATH = 'blink/intent_to_implement.html'
+
+  def process_post_data(self, **kwargs):
+    self.require_task_header()
+    feature_id = self.get_param('feature_id', required=True)
+    feature = FeatureEntry.get_by_id(feature_id)
+    if not feature:
+      self.abort(400, 'Feature not found.')
+    json_data = self.get_json_param_dict()
+    send_emails([self.build_email(feature, json_data)])
```

Comment:
It would be useful to first call build_email, then log some details of it, then call send_emails.  That way we have some clues in the logs for debugging.

Reply:
I've added some logging that logs the subject and recipients. The body contents get logged before submission (at least on staging), and I thought it might be long-winded to print the intent body in the logs again separately.

Reply:
OK